particles/Tutorial/Kotlin/Demo/src/TTTGame.kt

```diff
@@ -32,6 +32,66 @@ class TTTGame : AbstractTTTGame() {
         arrayOf(2, 4, 6)
     )
 
+    init{
+
+        handles.playerOneMove.onUpdate() { move ->
```

Comment:
It's more conventional to omit the parens: `onUpdate { move ->`

(Here and throughout file)